DeepSpeech.py

```diff
@@ -1150,6 +1159,24 @@ def _next_epoch(self):
         # Indicates, if there were 'new' epoch(s) provided
         result = False
 
+        if self._dev:
```

Comment:
So `self._dev` is used as a state machine flag. But then it should not get initialized by a user parameter.